game/src/environment/towers/core/Tower.gd

```diff
@@ -40,6 +43,7 @@ func _ready() -> void:
 	_reset()
 	# warning-ignore:return_value_discarded
 	_timer.connect("timeout", self, "_fire")
+	$Range/RangeRadius.shape.radius = range_radius 
```

Comment:
Shouldn't this be moved into` _reset()`

Reply:
either way its wrong. Think i have to relook at the tower range things.

Reply:
TC said handle later (on tele): https://github.com/quantum-defence/quantum-defence/issues/89